pages/components/cards.js

```diff
@@ -195,3 +278,37 @@ const LikeIconImage = styled.img`
   left: 50%;
   transform: translate(-50%, -50%);
 `;
+
+const DropdownButton = styled.button`
+  background-color: transparent;
+  margin-left: 10px;
+  padding: 8px;
+  border: none;
+  cursor: pointer;
+`;
+
+const DropdownContent = styled.div`
```

Comment:
can you find a better html element than div?

Reply:
change to li